StaticAnalyzer/models.py

```diff
@@ -2,20 +2,20 @@
 # Create your models here.
 
 class RecentScansDB(models.Model):
-    NAME = models.TextField()
+    NAME = models.CharField(max_length=200)

```

Comment:
What if the file has a really long name?

Reply:
I searched, the maximum file name allowed by the operating system is 255, we can set this to

https://stackoverflow.com/questions/265769/maximum-filename-length-in-ntfs-windows-xp-and-windows-vista
https://unix.stackexchange.com/questions/32795/what-is-the-maximum-allowed-filename-and-folder-size-with-ecryptfs